.github/pull_request_template.md

```diff
@@ -1,8 +1,6 @@
 ## Checks
 
 - [ ] I have read the [contributing guidelines](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/CONTRIBUTING.md).
-- [ ] If some rule is added/updated/removed, I've regenerated the rules list (`npm run generate:rules-list`)
-- [ ] If some rule meta info is changed, I've regenerated the plugin shared configs (`npm run generate:configs`)
```

Comment:
These aren't needed anymore since this is checked on CI.

Reply:
Wonderful! 🤩